scss/_patterns_strip.scss

```diff
@@ -1,9 +1,11 @@
 // Full width strip pattern
 @mixin vf-p-strip {
 
+  // strip
   %strip {
     background-color: transparent;
     clear: both;
+    margin-bottom: 2rem;
```

Comment:
Couldn't we just use `padding: 2rem 0;` for all strips? Then we wouldn't need to do the margin dance? Does the margin-bottom give us something I'm missing?

For example:
http://codepen.io/anthonydillon/pen/vgKKjw